src/page/form/Form.js

```diff
@@ -0,0 +1,208 @@
+import React, { Component } from 'react'
+
+const subject = [
+  "Đá bóng",
+  "Cầu lông",
+  "Đua thuyền"
+];
+
+export default class Form extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      txtname:"",
+      txtemail:"",
+      txtpass:"",
+      txtavatar:"",
+      txtphone:"",
+      txtage:1,
+      txtgender:"male",
+      txtnote:"Buổi học đầu tiên về HTML",
+      txthobbi: new Set()
+    };
+    this.handleChange =this.handleChange.bind(this);
+    this.handleSubmit =this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    var target = event.target;
+    var name = target.name;
+    var value = target.type === 'checkbox' ? target.checked : target.value;
+    this.setState({
+      [name]: value
+    });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+
+    var nhobbi = "";
+    for(const hb of this.state.txthobbi){
+      nhobbi =+ hb +",";
+    }
+
+    console.log(this.state);
+  }
+
+  handleCheckBox = () => {
+    const hobbi = subject.map((value,key) => {
+      return <label className="hobbi" key ={key}><input
+      type="checkbox"
+      name="txthobbi"
+      value={value}
+      onChange= {() => this.checkedSubject(value)}
+      checked = {this.state.txthobbi.has(value)}
+    />{value}</label>
+    });
+    return hobbi
+  }
+  componentWillMount() {
+    this.checkedCheckBox = new Set();
+  }
+
+  checkedSubject = (hobbi) => {
+    if(this.checkedCheckBox.has(hobbi)){
+      this.checkedCheckBox.delete(hobbi);
+    } else {
+      this.checkedCheckBox.add(hobbi);
+    }
+
+    this.setState({
+      txthobbi: this.checkedCheckBox
+    })
+    console.log(this.state.txthobbi)
+  }
+
+  handleFile = (event) => {
+    console.log(event.target.files[0].name)
+    this.setState({
+      txtavatar: event.target.files[0].name
+    });
+  }
+
+
+  render() {
+    return (
+      <>
+        <div className="container">
+          <div className="banner">
+            <h1>Đăng ký thành viên</h1>
+          </div>
+          <form class="form-horizontal" onSubmit={this.handleSubmit}>
```

Comment:
Mỗi khối có logic khác nhau thì hãy thêm 1 `enter`.
Các attributes/props của 1 element/component thì viết trên 1 dòng, không viết tất cả trên 1 dòng như thế
```suggestion
          <div className="banner">
            <h1>Đăng ký thành viên</h1>
          </div>

          <form
            class="form-horizontal"
            onSubmit={this.handleSubmit}>
```